### DISCUSSION TITLE
Lighting issue with architecture kit assets

### DISCUSSION THREAD

**Discussion by snieradka** [mozilla/hubs#2869] [General Questions & Discussion] [answered]
New to hubs; was making a scene in spoke with the architecture kit assets and sketchfab models. When I add a eleventh light source (I know hubs recommends 3, but performance is still acceptable in my scene on a 6 year old crappy laptop) all the architecture kit assets become invisible in shadow mode (they appear in lit view mode) , or when publish to hubs. Assets from sketchfab still exist normally in the scene.

**Comment by robertlong** [comment, marked as answer]
I believe Three.js has a maximum of 10 lights in a scene. I'm having a hard time finding the code responsible for this though.
